src/crypto/tls/auth.go

```diff
@@ -139,10 +146,12 @@ func typeAndHashFromSignatureScheme(signatureAlgorithm SignatureScheme) (sigType
 		hash = crypto.SHA512
 	case Ed25519:
 		hash = directSigning
-	case EdDilithium3:
-		hash = directSigning
 	default:
-		return 0, 0, fmt.Errorf("unsupported signature algorithm: %#04x", signatureAlgorithm)
+		scheme := circlSign.SchemeByTLSIdentifier(uint(signatureAlgorithm))
+		if scheme == nil {
+			return 0, 0, fmt.Errorf("unsupported signature algorithm: %#04x", signatureAlgorithm)
+		}
+		hash = directSigning
```

Comment:
All the schemes on circle will have directSigning, right?

Reply:
Yes, for the moment.